rollup.config.js

```diff
@@ -0,0 +1,58 @@
+import nodeResolve from 'rollup-plugin-node-resolve'
+import babel from 'rollup-plugin-babel'
+import replace from 'rollup-plugin-replace'
+import commonjs from 'rollup-plugin-commonjs'
+import { terser } from 'rollup-plugin-terser'
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot'
+import pkg from './package.json'
+
+const input = 'src/index.js'
+const name = 'styledSystem'
+
+const external = id => !id.startsWith('.') && !id.startsWith('/')
+
+const getBabelOptions = ({ useESModules }) => ({
+  exclude: '**/node_modules/**',
+  runtimeHelpers: true,
+  presets: [['@babel/preset-env', { loose: true, modules: false }]],
```

Comment:
You may remove this line since env plugin has always `modules: true` with rollup-plugin-babel.